index.bs

```diff
@@ -474,6 +462,59 @@ To perform a <dfn abstract-op>cross-origin resource policy check</dfn> given a [
         extensions, and I think it'll be more difficult to ship them after inverting the
         error-handling behavior.
 
+To perform a <dfn abstract-op>cross-origin resource policy check</dfn> given a [=request=]
+(|request|) and a [=response=] (|response|), run these steps:
+
+1.  Let |embedder policy| be |request|'s [=request/client=]'s
+    [=environment settings object/embedder policy=].
+
+2.  If |request|'s [=request/reserved client=] is not `null`, then set |embedder policy|
+    to a new [=/embedder policy=].
+
+3.  If |embedder policy|'s [=embedder policy/report only reporting endpoint=] is not `null` and the
+    result of running [$cross-origin resource policy internal check] with
+    [=embedder policy/report only value=], |request| and |response| is `blocked`, then run these
+    steps:
+
+    1. Let |blocked url| be the first URL of |request|'s [=request/URL list=].
+
+    2. Set |blocked url|'s [=url/username=] to the empty string, and its [=url/password=] to `null`.
+
+    3. Set |serialized blocked url| be the result of executing the
+       [URL serializer](https://url.spec.whatwg.org/#concept-url-serializer) on |blocked url| with
+       the |exclude fragment flag| set.
+
+    4. Let |body| be a new object containing the following properties with keys:
+
+        * key: "`blocked`", value: |serialized blocked url|.
```

Comment:
Is this enough context? It's not clear to me what @arturjanc and co. actually need (nor do I recall off the top of my head what the Reporting API provides on its own... Is https://w3c.github.io/reporting/#try-delivery up to date?).

Reply:
It looks like the default `url` value provided by the Reporting API and the `blocked` URL here would be sufficient for developers to debug CORP violations.

BTW, I'm not sure if it's intentional but CSP uses `blocked-uri` as the key for resources that didn't load (as opposed to `blocked` here), so maybe there's some value in making this consistent?

Reply:
Another value that could potentially be useful is `Request.destination` (so the developer would know what kind of resource didn't load, e.g. they could prioritize fixing scripts) but this is a more of a nice-to-have than a necessity for debugging.

Reply:
Nit: `blocked-uri` is deprecated. `blocked-url` (with an L, not an I) is preferred. :)

(I suspect @domenic would prefer that we follow the suggestions in https://w3ctag.github.io/design-principles/#casing-rules, using `_` instead of `-`. Unfortunately, CSP is old and crusty and predates those good ideas.)